### PR TITLE
Antag improvements part a lot

### PR DESCRIPTION
**Pull request by Stalkeros2**
# What this PR does
Adds a lot of items that should've been added a long time ago: revolutionary headset, radio, frequency and goggles, as well as improved rev officer coat sprite, improved antags' headsets sprite, slight changes to Slav-Safe helmet, fixed Slav-Safe armor positioning, a bunch of gas masks and something I forgot.

# Why it should be added to the game
More content good.

### CODE REVIEW

**Review by FancyCyclops — CHANGES_REQUESTED** [review]
bruh

**Review by Stalkeros2 — COMMENTED** [review]
(Cyclo is wrong.)

**Comment by Stalkeros2** [comment]
image

**Comment by BurgerLUA** [comment]
I really wish this was atomized. Please tell me when this is fine to merge.

File: code/_core/datum/damagetype/melee/sword/rev_bayonet.dm [hunk 1]
```diff
@@ -30,5 +30,13 @@
...
+	attack_delay = 6
+	attack_delay_max = 8
+
+	crit_multiplier = 4
+
+/damagetype/melee/sword/rev_bayonet/get_crit_chance(var/mob/living/L)
+	return 0
+
+/damagetype/melee/sword/rev_bayonet/get_critical_hit_condition(var/atom/attacker,var/atom/victim,var/atom/weapon,var/atom/hit_object)
+	return attacker.dir == victim.dir
```
Comment (BurgerLUA) [inline code/_core/datum/damagetype/melee/sword/rev_bayonet.dm:42]: Use ``get_true_4dir`` for both of these

File: code/_core/obj/item/tape.dm [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	size = SIZE_1
+	var/song_to_play = null
+
+/obj/item/tape/click_self(var/mob/caller,var/atom/object)
+
+	INTERACT_CHECK
+	INTERACT_DELAY(20)
+
+	if(icon_state == "inventory")
+		play_sound(song_to_play,get_turf(src),loop=TRUE)
```
Comment (BurgerLUA) [inline code/_core/obj/item/tape.dm:20]: Again the sound system can't really support moving music so this should only play to the caller and be disabled when the player removes it from their inventory.

File: code/_core/obj/item/weapon/ranged/spellgem/rift.dm [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+	shoot_sounds = list('sound/weapons/magic/heal.ogg')
+
+	color = "#FFD800"
+	color_2 = "#917E1F"
+	color_3 = "#7F0000"
+
+	value = 1800
+
+/obj/item/weapon/ranged/spellgem/rift/revive
```
Comment (BurgerLUA) [inline code/_core/obj/item/weapon/ranged/spellgem/rift.dm:26]: This is way too strong to be in the game. I don't want to subvert medicine.

File: config/admin.txt [hunk 4]
```diff
@@ -1,75 +0,0 @@
-# This is an example config file.
```
Comment (BurgerLUA) [inline config/admin.txt:1]: This file shouldn't be deleted.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

**Comment by Stalkeros2** [comment]
:ok_hand:

File: code/_core/datum/damagetype/melee/sword/rev_bayonet.dm (hunk 1 above)
Comment (Stalkeros2) [reply code/_core/datum/damagetype/melee/sword/rev_bayonet.dm:42]: wtf is get_true_4dir

File: code/_core/obj/item/tape.dm (hunk 2 above)
Comment (Stalkeros2) [reply code/_core/obj/item/tape.dm:20]: :pensive: Yeah it'll be unticked.

Comment (Stalkeros2) [reply code/_core/obj/item/tape.dm:20]: I think.

Comment (Stalkeros2) [reply code/_core/obj/item/tape.dm:20]: Oh wait what you've written actually works.

File: code/_core/obj/item/weapon/ranged/spellgem/rift.dm (hunk 3 above)
Comment (Stalkeros2) [reply code/_core/obj/item/weapon/ranged/spellgem/rift.dm:26]: Dam.

File: config/admin.txt (hunk 4 above)
Comment (Stalkeros2) [reply config/admin.txt:1]: It's an admin.txt file I've made to get myself admin stuff.

File: code/_core/obj/item/weapon/ranged/spellgem/rift.dm (hunk 3 above)
Comment (Stalkeros2) [reply code/_core/obj/item/weapon/ranged/spellgem/rift.dm:27]: Вщту

File: code/_core/datum/damagetype/melee/sword/rev_bayonet.dm (hunk 1 above)
Comment (Stalkeros2) [reply code/_core/datum/damagetype/melee/sword/rev_bayonet.dm:42]: It still works tho.

**Review by BurgerLUA — APPROVED** [review]

**Pull request merged**